pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -30,12 +30,14 @@ import (
 
 	"github.com/containernetworking/cni/libcni"
 	cnitypes "github.com/containernetworking/cni/pkg/types"
+	machinerytypes "k8s.io/apimachinery/pkg/types"
```

Comment:
this is a new alias for this import =). Consider using one of existing ones:

```
kubetypes "k8s.io/apimachinery/pkg/types"
k8stypes "k8s.io/apimachinery/pkg/types"
types "k8s.io/apimachinery/pkg/types"
commontypes "k8s.io/apimachinery/pkg/types"
```

also it is used so often so maybe worth placing into `hack/.import-aliases`